### PR TITLE
Issue 128

### PR DESCRIPTION
**Pull request by patrikspiess**

### CODE REVIEW

File: docs/source/developer/helpers/result.rst [hunk 1]
```diff
@@ -0,0 +1,256 @@
...
+
+    ┏━━━━━━━┳━━━━━━━━━━━━━┳━━━━━━━┳━━━━━━━━━━━━━┓
+    ┃ key   ┃ version     ┃ build ┃ host        ┃
+    ┡━━━━━━━╇━━━━━━━━━━━━━╇━━━━━━━╇━━━━━━━━━━━━━┩
+    │ host1 │ 1           │ 2000  │ host1.local │
+    │ host2 │ host2.local │ 2     │             │
+    │ host3 │ text        │       │             │
+    └───────┴─────────────┴───────┴─────────────┘
+
+You would expect a table with looks like the line for host1. But for host2 you recognize that the
```
Comment (lucmurer) [inline docs/source/developer/helpers/result.rst:209]: You would expect a table **which**...

File: docs/source/developer/helpers/result.rst [hunk 2]
```diff
@@ -0,0 +1,256 @@
...
+hostname is in the wrong column. This is because the order of the key value pairs int the dict does
```
Comment (lucmurer) [inline docs/source/developer/helpers/result.rst:210]: ... key value pairs **in** the dict ...

File: docs/source/developer/helpers/result.rst [hunk 3]
```diff
@@ -0,0 +1,256 @@
...
+not match the ordering of the first result. With host3 which not even has a dict als value you see
```
Comment (lucmurer) [inline docs/source/developer/helpers/result.rst:211]: ... not even has a dict **as** value ...

File: docs/source/developer/helpers/result.rst [hunk 4]
```diff
@@ -0,0 +1,256 @@
...
+Conclusion: **Stay consequent with dicts in results!**
+
+
+print_table_raw()
+-----------------
+
+The method :code:`print_table_raw()` takes the data to print as first argument. You always have to
+pass a list of dicts or it will raise an exception. Every item in the list will result in a line in
+the table and every key in the dict will represent a column. The table will only contain the values
+in the dicts whereas the keys wil be interpreted as colum headers. It is very important that every
```
Comment (lucmurer) [inline docs/source/developer/helpers/result.rst:223]: ... the keys **will** be ...

**Review by lucmurer — APPROVED** [review]
Some minor typos in the documentation, very well done!

File: docs/source/developer/helpers/result.rst (hunk 1 above)
Comment (patrikspiess) [reply docs/source/developer/helpers/result.rst]: fixed

File: docs/source/developer/helpers/result.rst (hunk 2 above)
Comment (patrikspiess) [reply docs/source/developer/helpers/result.rst]: fixed

File: docs/source/developer/helpers/result.rst (hunk 4 above)
Comment (patrikspiess) [reply docs/source/developer/helpers/result.rst]: fixed

File: docs/source/developer/helpers/result.rst (hunk 3 above)
Comment (patrikspiess) [reply docs/source/developer/helpers/result.rst]: fixed

**Review by lucmurer — APPROVED** [review]

**Pull request merged**